java-checks/src/main/java/org/sonar/java/checks/CollectorsToListCheck.java

```diff
@@ -38,7 +38,7 @@
 
 @Rule(key = "S6204")
 public class CollectorsToListCheck extends AbstractMethodDetection implements JavaVersionAwareVisitor {
-  private static final String MESSAGE = "Replace this usage of 'Stream.collect(Collectors.%s())' with 'Stream.toList()'";
```

Comment:
I realize it's not your doing, but this issue message does not follow the convention: https://docs.sonarsource.com/sonarqube/latest/extension-guide/adding-coding-rules/#messages

Issue message should finish by a dot. Can you fix this?